### PR TITLE
chore: correct hierarchy of BrowserWindow headings

### PR DESCRIPTION
**Pull request by erickzhao**
#### Description of Change

Reverts some of the changes in #19196 where two headings were moved up a level by accident.

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] relevant documentation is changed or added
- [x] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes

Notes: none

### CODE REVIEW

File: docs/api/browser-window.md [hunk 1]
```diff
@@ -24,10 +24,11 @@ you can use the [Frameless Window](frameless-window.md) API.
 
 ## Showing window gracefully
```
Comment (malept) [inline docs/api/browser-window.md:25]: This is tangential from the focus of this PR, but this and the other changed heading lines feel like they need definite articles:
```suggestion
## Showing the window gracefully
```

File: docs/api/browser-window.md [hunk 2]
```diff
@@ -24,10 +24,11 @@ you can use the [Frameless Window](frameless-window.md) API.
...
 
-When loading a page in the window directly, users may see the page load incrementally, which is not a good experience for a native app. To make the window display
-without visual flash, there are two solutions for different situations.
+When loading a page in the window directly, users may see the page load incrementally,
+which is not a good experience for a native app. To make the window display
+without a visual flash, there are two solutions for different situations.
 
-## Using `ready-to-show` event
+### Using `ready-to-show` event
```
Comment (malept) [inline docs/api/browser-window.md:31]:
```suggestion
### Using the `ready-to-show` event
```

File: docs/api/browser-window.md [hunk 3]
```diff
@@ -48,7 +49,7 @@ event.
 Please note that using this event implies that the renderer will be considered "visible" and
 paint even though `show` is false.  This event will never fire if you use `paintWhenInitiallyHidden: false`
 
-## Setting `backgroundColor`
+### Setting `backgroundColor`
```
Comment (malept) [inline docs/api/browser-window.md:52]:
```suggestion
### Setting the `backgroundColor` property
```

**Review by malept — APPROVED** [review]
My comments are non-blocking and can be pursued in a follow-up PR.

**Review by VerteDinde — APPROVED** [review]

**Review by zcbenz — APPROVED** [review]

**Pull request merged**